delta/app/src/main/resources/application-cassandra.conf

```diff
@@ -0,0 +1,71 @@
+include required(classpath("default.conf"))
+
+akka {
```

Comment:
There are already configuration files in sourcing.

It would be better to have them there and try to test these configuration files there no ?